patches/external-editor+3.1.0.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/node_modules/external-editor/main/errors/CreateFileError.js b/node_modules/external-editor/main/errors/CreateFileError.js
+index 7faa34c..7b49f68 100644
+--- a/node_modules/external-editor/main/errors/CreateFileError.js
++++ b/node_modules/external-editor/main/errors/CreateFileError.js
+@@ -14,7 +14,14 @@ var __extends = (this && this.__extends) || (function () {
+     };
+     return function (d, b) {
+         extendStatics(d, b);
+-        function __() { this.constructor = d; }
++        function __() {
++            Object.defineProperty(this, 'constructor', {
++                value: d,
++                writable: true,
++                enumerable: true,
++                configurable: true
++            });
++        }
+         d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
+     };
+ })();
+diff --git a/node_modules/external-editor/main/errors/LaunchEditorError.js b/node_modules/external-editor/main/errors/LaunchEditorError.js
+index 85a164e..217f192 100644
+--- a/node_modules/external-editor/main/errors/LaunchEditorError.js
++++ b/node_modules/external-editor/main/errors/LaunchEditorError.js
+@@ -14,7 +14,14 @@ var __extends = (this && this.__extends) || (function () {
+     };
+     return function (d, b) {
+         extendStatics(d, b);
+-        function __() { this.constructor = d; }
++        function __() {
++            Object.defineProperty(this, 'constructor', {
++                value: d,
++                writable: true,
++                enumerable: true,
++                configurable: true
++            });
++        }
+         d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
```

Comment:
After staring at this line for a bit, I have a vague notion of what it does. I think that's sufficient to approve this, but you might do well to wait for other reviewers who are more familiar with this sort of magic.